### ISSUE TITLE
valid-jsdoc incorrectly flags missing parameter

### ISSUE THREAD

**Issue by webOS101** [eslint/eslint#7555]
**Tell us about your environment**

* **ESLint Version:** 3.9.1
* **Node Version:** 6.8.0
* **npm Version:** 3.10.9

**What parser (default, Babel-ESLint, etc.) are you using?**  eslint-plugin-babel@3.3.0

**Please show your full configuration:**

```
{
	env: {
		'es6': true, // sets the "ecmaVersion" parser option to 6
	},
	parser: 'babel-eslint',
	plugins: [
		'babel'
	],
	rules: {
		'valid-jsdoc': 1
	}
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
/**
 * @class Whoops
 */
class Whoops {
	test = (wow) => {
		return wow;
	}
}

export default Whoops;
```

**What did you expect to happen?**  No warning, `test` does not have a jsdoc tag

**What actually happened? Please include the actual, raw output from ESLint.**
`  1:1  warning  Missing JSDoc for parameter 'wow'  valid-jsdoc`

If the method is changed to `test(wow) { }` then no warning is shown.

**Comment by vitorbal** [comment]
@webOS101 thank you for your issue. Just to be sure, could you check if you can reproduce the same problem using the default eslint parser instead of babel-eslint, and without the babel plugin?

**Comment by webOS101** [comment]
Well, I don't get a jsdoc warning, but instead get:
`  5:7  error  Parsing error: Unexpected token =`
Which is not so helpful.

**Comment by not-an-aardvark** [comment]
I think this issue is caused by your code's use of [class properties](https://github.com/tc39/proposal-class-public-fields), which is still an experimental syntax. ESLint's core rules generally don't support experimental features, so you may want to disable the rule in this case (e.g. with an inline comment: `// eslint-disable-next-line valid-jsdoc`).

You might also want to file an issue on [eslint-plugin-babel](https://github.com/babel/eslint-plugin-babel), a plugin containing rules that are modified to support experimental syntax.

**Comment by webOS101** [comment]
I suspected that would be the case.  I will go file an issue there, thanks!
